### PR TITLE
Use custom Slack notifications for WordPress develop builds

### LINKED ISSUE #4: Custom notifications to Slack
Circle's default Slack notifications aren't very helpful in our context:

image

It would be cool if we could indicate whether or not the build failed, and then the stats that PHPUnit reports about the test suite (# passed, #skipped, # failed)

### PR DESCRIPTION
**Pull request by danielbachhuber**
Fixes #4

### CODE REVIEW

**Comment by pantheon-circleci** [comment]
@danielbachhuber, thanks for your PR! By analyzing the blame information on this pull request, we identified @stevector to be a potential reviewer

**Comment by danielbachhuber** [comment]
@stevector Could I get an initial review on this?

File: cleanup.sh [hunk 1]
```diff
@@ -8,6 +8,17 @@ if [ -z "$TERMINUS_SITE" ] || [ -z "$TERMINUS_ENV" ]; then
 fi
 
 ###
+# Report the results to Slack (privately)
+###
+set +x
+if [ ! -z "$SLACK_WEBHOOK" ] && [ ! -z "$SLACK_CHANNEL" ]; then
+	PANTHEON_SSH_USER=$(terminus site connection-info --field=sftp_username)
+	PANTHEON_SSH_HOST=$(terminus site connection-info --field=sftp_host)
+	ssh -p 2222 -o StrictHostKeyChecking=no $PANTHEON_SSH_USER@$PANTHEON_SSH_HOST wp phpunit-report $SLACK_WEBHOOK $SLACK_CHANNEL $CIRCLE_BUILD_URL
```
Comment (danielbachhuber) [inline cleanup.sh:18]: @stevector Locally, this fails like so:

```
local ➜  pantheon-wordpress-develop git:(4-slack-notifs) ✗ ./cleanup.sh
+ '[' -z wordpress-develop ']'
+ '[' -z d-test-725 ']'
+ set +x
usage: wp phpunit-report <slack-webhook> <slack-channel> [<build-url>]
```

What am I doing wrong?

Comment (stevector) [inline cleanup.sh:18]: I'm not certain, @greg-1-anderson? Also, shouldn't this go in the `notify` step of CircleCI?

Comment (danielbachhuber) [inline cleanup.sh:18]: > Also, shouldn't this go in the notify step of CircleCI?

Notify [isn't a step](https://circleci.com/docs/configuration/#notify), but a configuration.

Comment (danielbachhuber) [inline cleanup.sh:18]: > What am I doing wrong?

It was escaping, of course f99dc80f66f36038166c70c070b61796f2873872

**Comment by stevector** [comment]
@danielbachhuber looks good to me. Anything you're concerned about or shall I merge? I suspect we may want to abstract some of the Slack posting logic as other repos need more complex reporting. But we can cross that bridge when we get there IMO.

**Comment by danielbachhuber** [comment]
> looks good to me.

Sweet :)

> Anything you're concerned about or shall I merge?

I'm pretty happy with it.

> I suspect we may want to abstract some of the Slack posting logic as other repos need more complex reporting. But we can cross that bridge when we get there IMO.

I agree. Eventually, all of this is worth abstracting. Not quite sure how yet.

**Pull request merged**